packages/core/networks/42161.json

```diff
@@ -22,5 +22,17 @@
   {
     "contractName": "OracleSpoke",
     "address": "0x196767568F848fF7607145A7C1f3CDE44bb4Ce98"
+  },
+  {
+    "contractName": "IdentifierWhitelist",
+    "address": "0x2914BE6CD56276d208299D2062848b059FC10588"
+  },
+  {
+    "contractName": "AddressWhitelist",
```

Comment:
I'm assuming we don't intend to have any automation for migrating collateral addresses and adding a final fee?

Do we have a script that could take the addresses and final fees as arguments and add them? Or do you think that's overkill? I only ask because we previously had an issue where we forgot to add initial final fees on polygon, causing free DVM requests to be possible.

Reply:
> Do we have a script that could take the addresses and final fees as arguments and add them?

We should add this but I'll save it for the next PR. Would be nice to be able to pipe in some list of addresses to whitelist all at once.

Reply:
Sounds good!